src/hist.jl

```diff
@@ -259,34 +272,42 @@ end
 append!{T,N}(h::AbstractHistogram{T,N}, vs::NTuple{N,AbstractVector}, wv::WeightVec) = append!(h, vs, values(wv))
 
 
+# Turn kwargs nbins into a type-stable tuple of integers:
+function _nbins_tuple{N}(vs::NTuple{N,AbstractVector}, nbins)
```

Comment:
The dispatch-based version consisted of two helper functions, wasn't that pretty, either, anyhow.